# EE3305/ME3243
# Name: Sim Justin
# NUSNET ID: E0968898

dt: 0.1

target_distance: 1.1
target_angle: 0.499

# Ku 14 Tu 2
Kp_f: 7
Ki_f: 1
Kd_f: 0.5

# Ku 10 Tu 1.8
Kp_a: 6
Ki_a: 1.8
Kd_a: 0.6

pillar_x: 7.5
pillar_y: 4